src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -182,30 +192,31 @@ else if (toRaise != null) {
     /**
      * @param file the file, may be null or not existing
      */
-    private void openTheFile(File file, boolean raisePanel) {
-        if ((file != null) && file.exists()) {
-            File fileToLoad = file;
-            frame.output(Localization.lang("Opening") + ": '" + file.getPath() + "'");
+    private void openTheFile(Path file, boolean raisePanel) {
+        if ((file != null) && Files.exists(file)) {
+            Path fileToLoad = file.toAbsolutePath();
+
+            frame.output(Localization.lang("Opening") + ": '" + file + "'");
 
-            String fileName = file.getPath();
-            Globals.prefs.put(JabRefPreferences.WORKING_DIRECTORY, file.getParent());
+            String fileName = file.getParent().toString();
```

Comment:
̀ fileName` is a bit irritating name here since it is actually the directory name.